ISISScientificComputing/autoreduce
# Fixes configuring new variables when there are only skipped runs

### Summary of work

Configuring new variables should work fine as long as there is a `reduce_vars.py`, as at no point are the previous run's variables used.

Changes the "See instrument variables" view to show "No current variables" if there are none.

Also adds test.

### How to test your work
Make sure to use these fixtures instead of the ones in the Quickstart page:
```
loaddata super_user_fixture status_fixture skipped_run
```

Follow the rest of the instructions from here to set up the rest: https://github.com/ISISScientificComputing/autoreduce/wiki/PR-testing-quickstart 

Fixes #857
